src/transformers/configuration_deberta.py

```diff
@@ -0,0 +1,124 @@
+# coding=utf-8
+# Copyright 2020, Microsoft
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" DeBERTa model configuration """
+
+from .configuration_utils import PretrainedConfig
+from .utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+DEBERTA_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "microsoft/deberta-base": "https://s3.amazonaws.com/models.huggingface.co/bert/microsoft/deberta-base/config.json",
+    "microsoft/deberta-large": "https://s3.amazonaws.com/models.huggingface.co/bert/microsoft/deberta-large/config.json",
+}
+
+
+class DeBERTaConfig(PretrainedConfig):
+    r"""
+    :class:`~transformers.DeBERTaConfig` is the configuration class to store the configuration of a
+    :class:`~transformers.DeBERTaModel`.
+
+    Arguments:
+        vocab_size (:obj:`int`, optional, defaults to 50265):
+            Vocabulary size of the DeBERTa model. Defines the different tokens that
+            can be represented by the `inputs_ids` passed to the forward method of :class:`~transformers.DeBERTaModel`.
```

Comment:
```suggestion
            can be represented by the :obj:`inputs_ids` passed to the forward method of :class:`~transformers.DeBERTaModel`.
```

Reply:
Even better, copy and adapt the docstring of the template/BERT